alfresco-search/src/main/java/org/alfresco/solr/tracker/MetadataTracker.java

```diff
@@ -104,14 +104,17 @@ public DocRouter getDocRouter() {
     @Override
     protected void doTrack() throws AuthenticationException, IOException, JSONException, EncoderException
     {
-
+        log.debug("### MetadataTracker doTrack ###");
```

Comment:
I'm not sure if you meant to include the debug logging or not (the PR comment says "without the debug statements").  If you do want to include them then I suggest we make them consistent with each other (e.g. if we want the hashes then it would be neater to have the same number everywhere).

Reply:
Yes we have another branch with all the additional logging, that we can merge at a later point.